src/evict/evict_lru.c

```diff
@@ -2363,11 +2363,11 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
          * Check if we've exceeded our operation timeout, this would also get called from the
          * previous txn is blocking call, however it won't pickup transactions that have been
          * committed or rolled back as their mod count is 0, and that txn needs to be the oldest.
-         *
-         * Additionally we don't return rollback which could confuse the caller.
          */
-        if (__wt_op_timer_fired(session))
+        if (__wt_op_timer_fired(session)) {
```

Comment:
@luke-pearson What's the concern of returning WT_ROLLBACK here? You add the comment in WT-6507.

Reply:
When `__wt_op_timer_fired` is executed in `__wt_txn_is_blocking` it returns `WT_ROLLBACK` to differentiate the two between this call was intended to not return `WT_ROLLBACK`. We discussed this with the execution side and they have made changes to handle it specifically.

Reply:
@luke-pearson I'm confused. Is it OK to return WT_ROLLBACK now?

Reply:
I don't think its okay to return `WT_ROLLBACK` here without further investigation / discussion with execution team.

Reply:
OK, I'll move it to another place then.